### ISSUE TITLE
Can't run Beta 0.0.7707.37574, .NET not detected, also breaks SCT.exe

### ISSUE THREAD

**Issue by BlohoJo** [AEAEAEAE4343/SimpleClassicThemeTaskbar#4]
OS is Win 10 Pro x64 build 2004.

Currently running SCT 1.1.9 and SCTT Alpha 0.0.7642.30810, working as expected.

The current directory structure for SCT and SCTT, is SCT is in C:\SCT, and SCTT is in C:\SCT\Taskbar. There is also a zip archive of SCTT which contains the files in C:\SCT\Taskbar in C:\SCT\sctt.zip.

I tried installing the new SCTT Beta 0.0.7707.37574 the same way. I moved the above C:\SCT\Taskbar folder and C:\SCT\sctt.zip file to a backup directory, then downloaded SimpleClassicThemeTaskbar_x64.zip to C:\SCT and renamed it to sctt.zip. I then extracted this to C:\SCT\Taskbar. So my setup looks like this:

![sctt_dir_main](https://user-images.githubusercontent.com/8242464/107135776-8ab99500-68cb-11eb-9c43-8a0b4d761bfe.png)

![sctt_dir](https://user-images.githubusercontent.com/8242464/107135777-8f7e4900-68cb-11eb-9897-39a903093e91.png)

I first tried running SCT.exe (still at version 1.1.9). I found that the "Enable" and "Disable" buttons are now greyed out:

![sct_options](https://user-images.githubusercontent.com/8242464/107135786-a58c0980-68cb-11eb-8773-a63f79a002fc.png)

If I move and replace C:\SCT\sctt.zip and C:\SCT\Taskbar\* with the older Alpha version 0.0.7642.30810, the above "Enable" and "Disable" buttons work again.

With the new SCTT Beta 0.0.7707.37574 folders back in place, I tried running SimpleClassicThemeTaskbar.exe directly. I received this error:

![sctt_net](https://user-images.githubusercontent.com/8242464/107135845-3b279900-68cc-11eb-87da-e1005e166fe7.png)

This dialog box disappears and does nothing regardless of which button, "Yes" or "No" is clicked.

I have Microsoft .NET installed up to 4.8. [ASoft .NET Version Detector](http://www.asoft.be/prod_netver.html) shows that I have installed Microsoft.NETCore.App 5.0.0.

![dotnet](https://user-images.githubusercontent.com/8242464/107135885-82ae2500-68cc-11eb-81a8-8a8ee557f84b.png)

Is there a specific installer I need for Microsoft .NET that I need in order to run SCTT Beta 0.0.7707.37574? If so, do you have a link to it?

Should I wait for a new version of SCT before I try running the new SCTT beta, since it appears that it breaks the current version 1.1.9? I see that SCT version 1.2.0 and 1.2.1 were posted but removed due to compatibility issues.

**Comment by BlohoJo** [comment]
I am trying to run the following installers for Microsoft .NET:

https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.2-windows-x86-installer
https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.2-windows-x64-installer

This worked fine on a Windows 7 x64 system.

On the Windows 10 system, I am being blocked by this error:

![dotnet5_windows10](https://user-images.githubusercontent.com/8242464/107136455-4e893300-68d1-11eb-9727-0c2b39f4636b.png)

I am running the installer as Administrator.

Can't find anyone else with the problem using Google search, so I guess I'm screwed. I hate Windows 10 so much.

**Comment by BlohoJo** [comment]
Ongoing troubleshooting:

- Ran [MicrosoftProgram_Install_and_Uninstall.meta.diagcab](https://support.microsoft.com/en-us/topic/fix-problems-that-block-programs-from-being-installed-or-removed-cca7d1b6-65a9-3d98-426b-e9f927e1eb4d), no problems found.

- Uninstalled x86 and x64 version of Microsoft .NET 5.0.

- Took ownership of C:\ProgramData folder.

- Deleted "BreadcrumbStore" folder under C:\ProgramData

Running the .NET 5.02 installers as Administrator, logged in as Administrator, User Account Control is disabled.

I am blocked by this:

![dotnet_block](https://user-images.githubusercontent.com/8242464/107136608-f2bfa980-68d2-11eb-99b4-46ad0ecf583d.png)

WTF

**Comment by BlohoJo** [comment]
Solved the installer issue, I deleted the C:\ProgramData\Microsoft\NetFramework folder. Now the installers ran. Again, WTF.

**Comment by BlohoJo** [comment]
Unfortunately, that didn't solve the problem of SCTT not being able to run. I still get:

![sctt_net](https://user-images.githubusercontent.com/8242464/107135845-3b279900-68cc-11eb-87da-e1005e166fe7.png)

Clicking "Yes" does nothing.

ASoft .NET Version Detector now says:

![dotnet_detect_update](https://user-images.githubusercontent.com/8242464/107136842-a9705980-68d4-11eb-9578-f6feac23446b.png)

**Comment by AEAEAEAE4343** [comment]
The zip package is not compatible with the old way SCT used to work. As you might’ve noticed SCT has switched to .NET 5.0. This means that the way the executables compile has changed: There’s now a EXE and a DLL file that both need to be called SimpleClassicThemeTaskbar. Also the resources have moved to separate files and even directories. This is also why the SCT+T public beta has been postponed indefinitely.

**Comment by AEAEAEAE4343** [comment]
That was an accident

**Comment by BlohoJo** [comment]
So then there's no way to get the posted SCTT Beta 0.0.7707.37574 to run with SCT 1.1.2 or 1.1.9?

**Comment by AEAEAEAE4343** [comment]
There is currently no way to get it to work at all

**Comment by BlohoJo** [comment]
OK, I understand! Please don't give up. I know I'm not the only user that considers what your working on to be a pretty big deal. A one person development team needs time! :)

**Comment by AEAEAEAE4343** [comment]
Delete C:\SCT\Taskbar and open the SCT options menu again
SCTT will reinstall and work correctly
